owltorrent-network/src/main/java/edu/rice/owltorrent/network/TorrentManager.java

```diff
@@ -415,4 +407,97 @@ private Bitfield buildBitfield() {
     }
     return bitfield;
   }
+
+  @Getter(AccessLevel.PACKAGE)
+  private final MessageHandler messageHandler =
```

Comment:
Does this make more sense as a method called handleMessage?

Reply:
So this was done to have PeerConnector no longer depend on TorrentManager. Which makes no sense from a design standpoint, but was done to save time... (Making it a method we'd still need to make a messageHandler to pass into peerConnectors, or we need to pass in TorrentManager)